sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { AppConfigCredential } from "./appConfigCredential";
+import { TokenCredential } from "@azure/core-http";
+import { ConfigurationClient } from "./generated/configurationClient";
+
+import * as Models from "./generated/models";
+
+const ConnectionStringRegex = /Endpoint=(.*);Id=(.*);Secret=(.*)/;
+const deserializationContentTypes = {
+  json: ["application/vnd.microsoft.appconfig.kvset+json"]
```

Comment:
I was testing the operations and found there are actually more content types than this we need to add!
Source: https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md
- application/vnd.microsoft.appconfig.kv+json
- application/vnd.microsoft.appconfig.kvs+json
- application/problem+json

Source: https://github.com/Azure/AppConfiguration/blob/master/docs/REST/keys.md
- application/vnd.microsoft.appconfig.keyset+json

Source: https://github.com/Azure/AppConfiguration/blob/master/docs/REST/revisions.md
- application/vnd.microsoft.appconfig.revs+json

Reply:
Oh joy!  Glad you caught those.